src/ui/castlejoysticks.pas

```diff
@@ -175,6 +177,11 @@   TJoysticks = class
       but it may be called in other cases (e.g. "explicit" joystick backend,
       used by Nintendo Switch, may call this at any moment). }
     property OnChange: TNotifyEvent read FOnChange write FOnChange;
+
+    { Called in case a previously initalized joystick had been unplugged unexpectedly. }
+    property OnDisconnect: TNotifyEvent read FOnDisconnect write FOnDisconnect;
```

Comment:
OnDisconnect could be of type `TSimpleNotifyEvent`, defined `TSimpleNotifyEvent = procedure of object;`